extensions/wikia/DesignSystem/DesignSystemHooks.class.php

```diff
@@ -12,9 +12,12 @@ class DesignSystemHooks {
 	 * @return bool true
 	 */
 	public static function onBeforePageDisplay( $out, $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$out". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)